.github/workflows/build.yml

```diff
@@ -185,60 +191,54 @@ jobs:
         with:
           submodules: true
       - uses: DataDog/action-prebuildify/test@main
+        if: matrix.node >= inputs.minimum-major-version
 
   linux-test:
+    strategy:
+      matrix:
+        node: [12, 14, 16, 18, 20]
     needs: linux-x64
     runs-on: ubuntu-latest
-    name: linux-test
+    name: linux-test-${{ matrix.node }}
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: true
-      - uses: DataDog/action-prebuildify/node/12@main
-      - uses: DataDog/action-prebuildify/test@main
-      - uses: DataDog/action-prebuildify/node/14@main
-      - uses: DataDog/action-prebuildify/test@main
-      - uses: DataDog/action-prebuildify/node/16@main
-      - uses: DataDog/action-prebuildify/test@main
-      - uses: DataDog/action-prebuildify/node/18@main
-      - uses: DataDog/action-prebuildify/test@main
-      - uses: DataDog/action-prebuildify/node/20@main
       - uses: DataDog/action-prebuildify/test@main
+        if: matrix.node >= inputs.minimum-major-version
```

Comment:
This should be on the job, not the step.

Reply:
It's not possible on the job. The if condition occurs before matrix construction.

Reply:
Can [exclude](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations) (or include) be used? It supports using data from the matrix.

Reply:
It's fine if this cannot be done at all, but given how slow Windows machines are, it would be good to not need to start more than we need if at all possible.